.github/workflows/actions/test-core-lint/action.yml

```diff
@@ -6,14 +6,12 @@ runs:
     - uses: actions/setup-node@v3
       with:
         node-version: 16.x
-
-    - name: Cache Core Node Modules
-      uses: actions/cache@v3
-      env:
-        cache-name: core-node-modules
-      with:
-        path: ./core/node_modules
-        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('./core/package-lock.json') }}-v2
+    - name: Install Dependencies
+      # Do not create a new package-lock file
+      # as it will cause the git diff check to fail.
+      run: npm install --no-save
```

Comment:
Can we use `npm ci` here or is there a reason we need to install the latest deps?

Reply:
Nope, we can use `npm ci`.